Makefile

```
@@ -171,7 +179,7 @@ kubecf-build: ##@kubecf Build chart from source and install KubeCF
 kubecf-clean: ##@kubecf Only delete installation of KubeCF & related files
 	$(MAKE) -C modules/kubecf clean
 
-.PHONY: sc	f
+.PHONY: scf
```

arg, this should have been

```suggestion
.PHONY: kubecf
```

seems that I introduced it long ago! hopefully it only created some `catapult/{sc,f}` files because of the phony. Sorry.